app/controllers/v0/education_benefits_claims_controller.rb

```diff
@@ -19,29 +19,6 @@ def create
       render(json: education_benefits_claim)
     end
 
-    # TODO: This is hidden behind a flag in the routes, but it's duplicated here for some
```

Comment:
The usage of the flag itself to disable the feature was duplicated, not the contents of the method itself.